advanced/Scripts/piholeLogFlush.sh

```diff
@@ -32,7 +37,7 @@ if [[ "$@" == *"once"* ]]; then
     # Nightly logrotation
     if command -v /usr/sbin/logrotate >/dev/null; then
         # Logrotate once
-        /usr/sbin/logrotate --force /etc/pihole/logrotate
+        /usr/sbin/logrotate --force --state $STATEFILE /etc/pihole/logrotate
```

Comment:
```suggestion
        /usr/sbin/logrotate --force --state "${STATEFILE}" /etc/pihole/logrotate
```

https://github.com/koalaman/shellcheck/wiki/SC2086

Needs to be quoted, and the `{}` are a style that we use.